README.md

```diff
@@ -55,6 +55,149 @@ Edit the file `cdc-accelerator.json` in the project directory and add the `sourc
 }
 ```
 
+### How to configure the use of features on the file cdc-accelerator.json
+
+On the cdc-accelerator.json file, there are two mandatory properties that the user has to fill, the "source" and "deploy".
+They both will have an array of objects that will contain the apiKeys that are related to the sites that we want to use in the project and optionally it will have the features.
+The features will be an optional array of strings that will have the name of the features that the user will want to have available to work on the specific site, for example:
+
+```
+{
+  "source":[
+    {
+    "apiKey":"1_B12AD0AISOPAD",
+    "features": ["Schema","PermissionGroup","WebSdk"]
+    }
+  ]
+}
+```
+
+In this example only the features "Schema","PermissionGroup" and "WebSdk" will be available for the apiKey "1_B12AD0AISOPAD".
+If the user doesn't provide the feature array it means that all of the features will be available and if it provides an empty array then no feature will be available, for example:
+
+```
+{
+  "source":[
+    {
+    "apiKey":"1_B12AD0AISOPAD",
+    "features": [""]
+    }
+  ]
+}
+```
+
+### How to use the feature specific commands
+
+Using the feature specific command lets the user run a specific feature (Schema, PermissionGroups, WebSdk, WebScreenSets,EmailTemplates...) instead of running all of them when doing an operation (init,reset,build,deploy).
+To use them, simply write on the terminal
+
+```
+npm run <operation> -- -f <feature>
+```
+
+For example:
+
+```
+npm run init -- -f Schema
+```
+
+In this example the user is only going to run the feature Schema when running the operation Init, the feature name can be replaced by any other feature (Email Templates, WebScreenSet, PermissionGroup, WebSdk...).
+To show all the possible commands, the user can write simply
+
+```
+npx cdc help
+```
+
+This command will show all possible commands and options that the user can do, the output will be something like this:
+
+```
+Usage: npx cdc [options] [command]
+
+A development environment for SAP Customer Data Cloud that enables the use of modern tools, such as JavaScript and source control.
+
+Options:
+  -V, --version     output the version number
+  -h, --help        display help for command
+
+Commands:
+  start             Launch local server for testing using the preview functionality
+  setup             Setup a new project after this dependency is installed
+  init [options]    Reads the data from the cdc console of the sites configured
+  reset [options]   Deletes the local folder and reads the data from the cdc console of the sites configured
+  build [options]   Processes the local data and prepares it to be deployed to the cdc console
+  deploy [options]  Deploys the local data to the cdc console on the sites configured
+  help [command]    display help for command
+```
+
+### How to use filters on preview
```

Comment:
This section needs a context, what is preview?
Where the filter is applied? (file src/index.html)